R/nSequences.R

```diff
@@ -0,0 +1,142 @@
+#' @title Count the number of identifications per scan
+#'
+#' @description
+#'
+#' The function takes a proteomics `Spectra` object containing
+#' identification results as input. It then counts the number of
+#' identifications each scan (or their descendants) has lead to - this
+#' is either 0 or 1 for MS2 scans, or, for MS1 scans, the number of
+#' MS2 scans originating from any MS1 peak that lead to an
+#' identification.
+#'
+#' This function can be used to generate id-annotated total ion
+#' chromatograms, as can illustrated
+#' [here](https://rformassspectrometry.github.io/docs/sec-id.html#an-identification-annotated-chromatogram).
+#'
+#' @details
+#'
+#' The computer number of identifications is stored in a new spectra
```

Comment:
`computed number` not `computer number`

Reply:
thanks, fixed in #58a6e50b0e63e849e0d0fd6ca76aff238b125678